api/lib/domain/usecases/get-assessment.js

```diff
@@ -49,7 +50,8 @@ async function _addCertificationRelatedAttributes(assessment, liveAlertRepositor
   if (assessment.type === Assessment.types.CERTIFICATION) {
     const liveAlerts = await liveAlertRepository.getByAssessmentId(assessment.id);
 
-    assessment.hasOngoingLiveAlert = liveAlerts.length > 0;
+    assessment.hasOngoingLiveAlert =
+      liveAlerts.filter((liveAlert) => liveAlert.status === CertificationChallengeLiveAlertStatus.ONGOING).length > 0;
```

Comment:
On cherche à savoir s'il y en a au moins 1, un some suffirait
```suggestion
      !liveAlerts.some((liveAlert) => liveAlert.status === CertificationChallengeLiveAlertStatus.ONGOING);
```